pkg/aws/services/vpclattice.go

```diff
@@ -20,6 +22,29 @@ type ServiceNetworkInfo struct {
 	SvcNetwork vpclattice.ServiceNetworkSummary
 	Tags       Tags
 }
+
+type LatticeServiceNameProvider interface {
+	LatticeServiceName() string
+}
+
+type NotFoundError struct {
+	ResourceType string
+	Name         string
+}
+
+func (e *NotFoundError) Error() string {
+	return fmt.Sprintf("%s %s not found", e.ResourceType, e.Name)
+}
```

Comment:
Do we need to change to:
```
return fmt.Sprintf("Lattice %s %s not found", e.ResourceType, e.Name)
```

So that log can show  "Lattice service xxxxxx not found" and "Lattice service network xxxxxx not found"

Reply:
I'll update this for next PR